tasks/install_system.yml

```diff
@@ -0,0 +1,68 @@
+- name: Make sure iscsi device for {{ vm.host }} @{{ vm.org }} is logged in
+  systemd:
+    name: vm_iscsi@{{ vm.org }}/{{ vm.host }}.service
+    state: started
+  when: vm.storage_type|default(xen_vman_storage_type) == "iscsi"
+
+- name: Format root storage for {{ vm.host }} @{{ vm.org }}
+  shell: mkfs.{{ vm.filesystem|default(xen_vman_default_filesystem) }} "/dev/disk/by-path/ip-{{ xen_vman_iscsi_server }}-iscsi-{{ xen_vman_iscsi_base_wwn }}:{{ vm.org }}-{{ vm.host }}-lun-0"
```

Comment:
That hardcoded lun-0.... Just out of interest, can I have multiple LUNs?

Reply:
Yes you can under special circumstances.